### PR TITLE
Text expressing the lack of consensus for send order and priorities.

### PR DESCRIPTION
**Pull request by huitema**
During the last editors meeting, we decided to not try to resolve the discussions on congestion and priorities
before draft adoption, and I was asked to propose text to that effect. This PR adds text expressing our lack of consensus on send order versus priorities.

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

**Review by suhasHere — APPROVED** [review]
this looks like a good start on this topic and we should merge it.

one small nit, should we move sendOrder and priorities under Section 7 ?

File: draft-lcurley-warp.md [hunk 1]
```diff
@@ -367,7 +367,24 @@ A media object:
...
+
+* Ensure that media objects are delivered in the order intended by the emitter
+* Allow nodes and relays to skip or delay some objects to deal with congestion
+* Ensure that emitters can accurately predict the behavior of relays
+* Ensure that when relay have to skip and delay objects belonging to different
+  tracks they do it in a predictable way if tracks are explicitly coordinated
+  and in a fair way if they are not.
+
+The working group has been considering two alternatives: mark objects belonging to a track
+with and explicit "send order"; and, define algorithms combining tracks, priorities and object
```
Comment (afrind) [inline draft-lcurley-warp.md:383]: an explicit

File: draft-lcurley-warp.md [hunk 2]
```diff
@@ -386,6 +403,29 @@ A receiver MUST NOT assume that objects will be received in send order for a num
...
+Media is produced as a set of layers, such as for example low definition and high definition,
+or low frame rate and high frame rate. Each media object belonging to a track and a group has two attributes: the object-id, and the priority (or layer).
+
+When nodes or relays have to choose which object to send next, they apply the following rules:
+
+* within the same group, objects with a lower priority number (e.g. P1) are always sent
+  before objects with a numerically greater priority number (e.g., P2)
+* within the same group, and the same priority level, objects with a lower object-id are
+  always sent before objects with a higher object-id.
+* objects from later groups are normally always send
```
Comment (afrind) [inline draft-lcurley-warp.md:417]: always sent

File: draft-lcurley-warp.md (hunk 1 above)
Comment (huitema) [reply draft-lcurley-warp.md]: Fixed.

File: draft-lcurley-warp.md (hunk 2 above)
Comment (huitema) [reply draft-lcurley-warp.md]: Fixed.

**Comment by huitema** [comment]
@suhasHere I tried to keep the PR minimal -- minimal text so the draft can be adopted without waiting for a consensus on handling of congestion.  I think that we can worry about moving sections around after the draft is adopted.

File: draft-lcurley-warp.md [hunk 3]
```diff
@@ -386,6 +403,29 @@ A receiver MUST NOT assume that objects will be received in send order for a num
 
 TODO: Refer to Congestion Response and Priorirization Section for further details on various proposals.
```
Comment (kixelated) [inline draft-lcurley-warp.md:404]: Remove this TODO?

**Review by kixelated — APPROVED** [review]

**Pull request merged**